apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/BackupStep.tsx

```diff
@@ -0,0 +1,235 @@
+    const [pathname, search] = recoverActivatePath.split("?");
+    dispatch(
+      saveSettings({
+        hasCompletedOnboarding: true,
+      }),
+    );
+    history.push({
+      pathname,
+      search: search ? `?${search}` : undefined,
+      state: { fromOnboarding: false },
+    });
+  }, [dispatch, history, recoverActivatePath]);
+
+  return isOpened ? (
+    <Flex rowGap={3} flexDirection={"column"}>
+      <ChoiceText mb={3}>{t("syncOnboarding.manual.backup.backupChoice.description")}</ChoiceText>
+      <ButtonV3
+        size="small"
+        variant="main"
+        onClick={navigateToRecover}
+        event="button_clicked"
+        eventProperties={{ button: "Try for free", flow: "Device onboarding" }}
+      >
+        {t("syncOnboarding.manual.backup.backupChoice.tryCta")}
+      </ButtonV3>
+      <ButtonV3
+        size="small"
+        variant="shade"
+        onClick={navigateToRecover}
+        event="button_clicked"
+        eventProperties={{ button: "Apply your redeem code", flow: "Device onboarding" }}
+      >
+        {t("syncOnboarding.manual.backup.backupChoice.redeemCodeCta")}
+      </ButtonV3>
+      <VideoLink />
+    </Flex>
+  ) : null;
+};
+
+const KeepOnPaperBody: React.FC<ChoiceBodyProps> = ({ isOpened, onPressKeepManualBackup }) => {
+  const { t } = useTranslation();
+  return isOpened ? (
+    <Flex flexDirection={"column"} mt={3}>
+      <ChoiceText mb={6}>{t("syncOnboarding.manual.backup.manualBackup.description")}</ChoiceText>
+      <ButtonV3
+        size="small"
+        variant="main"
+        onClick={onPressKeepManualBackup}
+        event="button_clicked"
+        eventProperties={{ button: "Keep it on Paper", flow: "Device onboarding" }}
+      >
+        {t("syncOnboarding.manual.backup.manualBackup.cta")}
+      </ButtonV3>
+    </Flex>
+  ) : null;
+};
+
+const WrappedTag: React.FC<{ text: string } & ComponentProps<typeof Tag>> = ({
+  text,
+  ...props
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Tag
+      size="tiny"
+      active
+      {...props}
+      textProps={{ uppercase: true, fontWeight: "semiBold", lineHeight: "normal" }}
+    >
+      {t(text)}
+    </Tag>
+  );
+};
+
+const choices: Choice[] = [
+  {
+    id: "backup",
+    getTitle: selected =>
+      selected
+        ? "syncOnboarding.manual.backup.backupChoice.titleSelected"
+        : "syncOnboarding.manual.backup.backupChoice.title",
+    icon: <Icons.ShieldCheck size={"S"} color="primary.c80" />,
+    tag: (
+      <WrappedTag
+        text={"syncOnboarding.manual.backup.backupChoice.tag"}
+        type="plain"
+        active
+        bg="primary.c80"
+      />
+    ),
+    body: BackupBody,
+  },
+  {
+    id: "keep_on_paper",
+    getTitle: () => "syncOnboarding.manual.backup.manualBackup.title",
+    icon: <Icons.Note size={"S"} color="neutral.c80" />,
+    body: KeepOnPaperBody,
+  },
+];
+
+const BackupStep: React.FC<Props> = props => {
+  const { device, onPressKeepManualBackup } = props;
```

Comment:
```suggestion
const BackupStep: React.FC<Props> =({ device, onPressKeepManualBackup }) => {
```
In addition, I think that it's better to rename the props by `BackupStepProps` to avoid a common `Props` as there is more than one component in the same file.